app/models/user.rb

```diff
@@ -1,16 +1,13 @@
 class User < ApplicationRecord
-  attr_accessor :remember_token, :session_token, :activation_token,
-                :reset_token, :from_google_oauth
-
-  has_secure_password
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
+  attr_accessor :from_google_oauth
```

Comment:
```suggestion
  devise :database_authenticatable, :registerable,
         :validatable, :confirmable
```
Pull này xử lý cho sign_up, e giới hạn tính năng lại nhé. Tránh thừa